exercises/clock/clock_test.go

```diff
@@ -51,6 +51,13 @@ func TestAddMinutes(t *testing.T) {
 				a.h, a.m, a.a, got, a.want)
 		}
 	}
+	// TODO: Don't duplicate.
```

Comment:
Not clear on the "duplicate" meaning.  The subtractTests are part of testing Add, so all seems ok here.